cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -53,7 +53,8 @@ def _create_course_with_access_groups(self, course_location, user=None):
         course = CourseFactory.create(
             org=course_location.org,
             number=course_location.course,
-            run=course_location.run
+            run=course_location.run,
+            default_store=ModuleStoreEnum.Type.mongo
```

Comment:
Tests verify operation counts for old mongo. Should be extended to split in the future.